lib/ldap.js

```diff
@@ -28,25 +34,37 @@ function initializeConnection () {
   });
 
   connection.heartbeat = function (callback) {
-    connection.search(nconf.get('LDAP_BASE'), nconf.get('LDAP_HEARTBEAT_SEARCH_QUERY'), function (err, res) {
-      if (err) {
+    // alway re-bind the connection in an heartbeat before searching, underlying connection might be bound to a different users from previous
```

Comment:
Mostly the same function, exept a bind is performed on the connection before attempting the search query. This fixes IUM-1416, where a previous login validation left the connection assigned to a user wihtout the same permissions to execute the heartbeat search query.